.github/workflows/release.yml

```
@@ -93,6 +92,21 @@ jobs:
           asset_name: ${{ runner.os }}.tar.gz
           asset_content_type: application/gzip
 
+      - name: Package Checksum
+        shell: bash
+        run: ./scripts/checksum ${{ runner.os }}.tar.gz ${{ runner.os }}.sha1
+
+      - name: Upload Checksum Asset
+        id: upload-checksum-asset
+        uses: actions/upload-checksum-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ needs.draft_release.outputs.upload_url }}
+          asset_path: ./${{ runner.os }}.sha
+          asset_name: ${{ runner.os }}.sha
```

In the previous step we point the script to a file with a `sha1` extension, so I think we should use that here too:
```suggestion
          asset_path: ./${{ runner.os }}.sha1
          asset_name: ${{ runner.os }}.sha1
```